python/hail/representation/genomeref.py

```diff
@@ -66,21 +65,26 @@ def __init__(self, name, contigs, lengths, x_contigs=[], y_contigs=[], mt_contig
         self._x_contigs = x_contigs
         self._y_contigs = y_contigs
         self._mt_contigs = mt_contigs
-        self._par = par
+        self._par = None
+        self._par_tuple = par
 
         super(GenomeReference, self).__init__()
 
+    def _set_history(self, history):
+        assert not self._history_was_set, "Cannot set history for GenomeReference more than once."
```

Comment:
sorry, another note:

`self._history_was_set` doesn't appear to get set here. Is it supposed to get set to True when this is called?

Reply:
good catch.